tycho-versions-plugin/src/main/java/org/eclipse/tycho/versions/engine/Versions.java

```diff
@@ -17,9 +17,9 @@
 import org.osgi.framework.Version;
 
 public class Versions {
-    private static final String SUFFIX_QUALIFIER = ".qualifier";
+    public static final String SUFFIX_QUALIFIER = ".qualifier";
```

Comment:
We probably should move those constants to the [TychoConstants](https://github.com/eclipse-tycho/tycho/blob/master/tycho-api/src/main/java/org/eclipse/tycho/TychoConstants.java) class.

Reply:
I can do this in another PR. Would you backport this PR to 4.0.x first?

Reply:
Just add it to this pr no need for several ones. It would also be good to improve the commit message a bit (e.g. describe what is the problem and what was changed).

Reply:
Will do.